flutter/example/lib/isar/user.g.dart

```diff
@@ -13,17 +13,17 @@ extension GetUserCollection on Isar {
   IsarCollection<User> get users => this.collection();
 }
 
-final UserSchema = CollectionSchema(
+const UserSchema = CollectionSchema(
   name: r'User',
-  id: BigInt.parse("-7838171048429979076").toInt(),
+  id: -7838171048429979076,
```

Comment:
@denrase the new isar generation doesn't work on web, see CI failure

this LoC seems to be problematic

Reply:
It looks like web should work with isar community 4.x

Reply:
think we had a some issue before, let me check

Reply:
should be fixed, same approach as before